packages/lib-sourcify/src/lib/CheckedContract.ts

```diff
@@ -20,10 +25,10 @@ const FETCH_TIMEOUT = parseInt(process.env.FETCH_TIMEOUT || '') || 3000; // ms
  */
 export class CheckedContract {
   /** Object containing contract metadata keys and values. */
-  metadata: Metadata;
+  metadata!: Metadata;
```

Comment:
In TypeScript, the exclamation mark ! after a property name is called a non-null assertion operator. It is used to tell the TypeScript compiler that a property or variable is not null or undefined, even though the compiler might think otherwise.

Here's an example:

```typescript
class Person {
  name!: string;
  age!: number;
}
```

In this example, the ! after name and age tells the compiler that these properties will be initialized at some point, even though they are not initialized in the constructor or anywhere else in the class. This is useful when dealing with situations where the TypeScript compiler can't infer whether a value is null or undefined, such as when the value is set asynchronously.

Note that using the non-null assertion operator should be done with care, as it can lead to runtime errors if the value is actually null or undefined. It is always better to check if a value is null or undefined before using it, instead of relying on the non-null assertion operator.